next.json

```diff
@@ -762,6 +762,13 @@
                     {"name": "output attachment location", "type": "uint32_t"},
                     {"name": "attachment slot", "type": "uint32_t"}
                 ]
+            },
+            {
+                "name": "subpass set depth stencil attachment",
```

Comment:
As we talked about in Hangouts, D3D12 only allows one resource for depth and/or stencil, so we can go with a single entry point.